AWS/legos/aws_filter_ec2_without_lifetime_tag/aws_filter_ec2_without_lifetime_tag.py

```diff
@@ -39,20 +39,22 @@ def aws_filter_ec2_without_lifetime_tag(handle, lifetime_tag: str, region: str)
         :rtype: Array of instances which not having lifetime tag.
     """
 
+    untagged_instances_dict={}
     ec2Client = handle.client('ec2', region_name=region)

```

Comment:
Are you checking if region is `''` here? Does handle.client take region_name as empty quotes?

Reply:
Oh. It doesn't. Maybe I can just give an empty List in the runbook itself and remove  `' '`

Reply:
Made the changes. Please re-review